_posts/2021-09-29-solidity-0.8.9-release-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.9 Release Announcement'
+date: '2021-09-29'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.9](https://github.com/ethereum/solidity/releases/tag/v0.8.9) fixes two bugs:
+
+ - one related to [immutables of signed integer types shorter than 256 bits](TODO LINK)
```

Comment:
apart from the link also add "introduced in version 0.6.5". This makes it similar in structure to the following summary of the user-defined value type bug.